dockerbuild/ssp-overrides/config.php

```diff
@@ -43,6 +70,31 @@
      */
      'hubmode' => $HUB_MODE,
 
+     /*
+      * Name of this IdP
+      */
+     'idp_name' => $IDP_NAME,
+
+     /*
+      * Name of this IdP to display to the user
+      */
+     'idp_display_name' => $IDP_DISPLAY_NAME,
+
+     /*
+      * The tracking Id for Google Analytics or some other similar service
+      */
+     'analytics.trackingId' => $ANALYTICS_ID,
+
+    /*
+     * Get a string of html to show as an announcement on the discovery page
+     * and/or login page.  By default, this will be fetched from
+     *   .../vendor/simplesamlphp/simplesamlphp/announcement/announcement.php
+     */
+    'announcement' => AnnouncementUtils::getAnnouncement(),
+    
+    'passwordChangeUrl' => $PASSWORD_CHANGE_URL,
```

Comment:
with these vars defined here, do they also need to be in `saml20-idp-hosted.php`? Or did we just put it in there in our appsdev instance for testing?

Reply:
I don't know. Perhaps @Baggerone does.

Reply:
This is my current understanding. The idp-hosted metadata includes an authproc call to the expirychecker.  That authproc needs to be told what the passwordChangeUrl is by the idp-hosted.php file.  It is currently doing that using Env::requireEnv('PASSWORD_CHANGE_URL').  

The PASSWORD_FORGOT_URL isn't used there.